Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -636,12 +590,23 @@ private extension SPNoteListViewController {
     ///
     func prefixText(for note: Note) -> String? {
         guard case .searching = notesListController.state,
-            let date = note.date(for: notesListController.searchSortMode)
+            let date = note.date(for: notesListController.sortMode)
             else {
                 return nil
         }
 
-        return DateFormatter.listDateFormatter.string(from: date)
+        return dateSortPrefix() + DateFormatter.listDateFormatter.string(from: date)
+    }
+
+    private func dateSortPrefix() -> String {
+        switch notesListController.sortMode {
+        case .createdNewest, .createdOldest:
+            return "Created: "
+        case .modifiedNewest, .modifiedOldest:
+            return "Modified: "
```

Comment:
(Although this code is going away!) Please always remember to NSLocalize all the things 😄